comixed-model/src/main/java/org/comixedproject/model/comicbooks/Comic.java

```diff
@@ -368,6 +368,8 @@
   private Date lastModifiedOn = new Date();
 
   @OneToOne(mappedBy = "comic", cascade = CascadeType.ALL, orphanRemoval = true)
+  @JsonProperty("metadataSource")
+  @JsonView({View.ComicListView.class, View.AuditLogEntryDetail.class})
   @Getter
   @Setter
   private ComicMetadataSource metadataSource;
```

Comment:
I went back and forth on this one. I'd prefer not to call the field "comicMetadataSource" since we know it's for the comic given it's parent object. Maybe we just simplify the name to "metadata", not as the container for the metadata but as the link between Comic and MetadataSource?

Reply:
I agree with this

Reply:
Okay, this change is made and pushed.